### DISCUSSION TITLE
如何修改创建表格的默认行列

### DISCUSSION THREAD

**Discussion by dingxiaoxue** [wangeditor-team/wangEditor#3957] [Q&A]
<img width="403" alt="image" src="https://user-images.githubusercontent.com/25151244/158755191-1176cdbd-f4f5-42a1-b7fc-f5b487bc0e51.png">

**Comment by wangfupeng1988** [comment]
现在不支持。建议去使用 v5 https://www.wangeditor.com/v5/
